db/seeds/source_types.rb

```diff
@@ -145,4 +145,39 @@ def update_or_create(attributes)
 update_or_create(:name => "ovirt", :product_name => "Red Hat Virtualization", :vendor => "Red Hat")
 update_or_create(:name => "openstack", :product_name => "Red Hat OpenStack", :vendor => "Red Hat")
 update_or_create(:name => "cloudforms", :product_name => "Red Hat CloudForms", :vendor => "Red Hat")
-update_or_create(:name => "satellite", :product_name => "Red Hat Satellite", :vendor => "Red Hat")
+
+satellite_json_schema = {
+  :authentication => [
+    {
+      :type   => "receptor_node",
+      :name   => "Receptore node",
+      :fields => [
+        {
+          :component         => "text-field",
+          :name              => "authentication.authtype",
+          :hideField         => true,
+          :initializeOnMount => true,
+          :initialValue      => "receptor_node"
+        },
+        {
+          :component  => "text-field",
+          :name       => "authentication.username",
```

Comment:
I'm thinking that this is not the userid but instead is an id the references the Satellite instance in the on prem env. I'll check on that.

Reply:
If the satellite ID is the unique identifier for the satellite we have `source.source_ref` for that

Reply:
Thanks Adam, Richard and I are discussing it now. I'll find out for sure and post here

Reply:
@rvsia @agrare I've confirmed that the Satellite ID that we need to store with the sources is a GUID that is used to identify an instance of a Satellite running on prem. So `source.source_ref` would be the place to save it.